### PR TITLE
Solves issue download progress bar issue

### PR DESCRIPTION
**Pull request by amsuhane**
Solves [poliastro/issues/542](https://github.com/poliastro/poliastro/issues/542).
The download progress bar looks like this:
astroquery_dastcom_downloadbar
Please let me know if there are further changes to be made.

### CODE REVIEW

File: astroquery/dastcom5/core.py [hunk 1]
```diff
@@ -58,16 +60,20 @@ def download_dastcom5(self):
...
+                urllib.request.urlretrieve(self.ftp_url + "dastcom5.zip", filename=dastcom5_zip_path, reporthook=t.update_to)
+
         with zipfile.ZipFile(dastcom5_zip_path) as myzip:
             myzip.extractall(self.local_path)
 
-    def _show_download_progress(self, transferred, block, totalsize):
-        trans_mb = transferred * block / (1024 * 1024)
-        total_mb = totalsize / (1024 * 1024)
-        print("%.2f MB / %.2f MB" % (trans_mb, total_mb), end="\r", flush=True)
+    class DownloadProgressBar(tqdm):
```
Comment (shreyasbapat) [inline astroquery/dastcom5/core.py]: Can we keep this as `Show_Download_Progress` ? And maybe making it private?

File: astroquery/dastcom5/core.py [hunk 2]
```diff
@@ -38,6 +39,7 @@ def __init__(self, spk_id=None, name=None):
         self.com_dtype = conf.COM_DTYPE
         self.ast_dtype = conf.AST_DTYPE
         self.ftp_url = conf.FTP_DB_URL
+        self.downloadprogressbar = self.DownloadProgressBar
```
Comment (shreyasbapat) [inline astroquery/dastcom5/core.py]: This class variable must also be private

File: astroquery/dastcom5/tests/test_dastcom5.py [hunk 3]
```diff
@@ -95,7 +95,7 @@ def test_download_dastcom5_downloads_file(mocker):
     mock_zipfile.is_zipfile.return_value = False
     Dastcom5.download_dastcom5()
     mock_request.assert_called_once_with(
-        Dastcom5.FTP_DB_URL + "dastcom5.zip",
-        os.path.join(Dastcom5.local_path, "dastcom5.zip"),
-        Dastcom5._show_download_progress,
+        Dastcom5.ftp_url + "dastcom5.zip",
```
Comment (shreyasbapat) [inline astroquery/dastcom5/tests/test_dastcom5.py]: Thanks for this :)

**Comment by amsuhane** [comment]
The build seems to be failing due to ``ModuleNotFoundError: No module named 'pandas'``. I am not sure why it is doing so. Could you guide me on how to correct this? Also please let me know if there are any other changes to be made.

**Comment by shreyasbapat** [comment]
Please don't worry about that! That is because, pandas is conventionally not a dependency of astroquery, hence not installed in CI. 
I would ask you to squash the commits into 1 or maximum of 2 :) Nothing else.

**Comment by shreyasbapat** [comment]
And do not forget to configure your local git! :) 
(You may want to configure user.name  and user.email)

**Comment by amsuhane** [comment]
Sorry for the mess :(  I have made the final change.

**Comment by shreyasbapat** [comment]
Great! @amsuhane Thanks for the valuable contribution! 
cc @Juanlu001

**Comment by shreyasbapat** [comment]
@amsunane , can you let me know the email id you use for git?

**Comment by shreyasbapat** [comment]
@amsuhane  , can you let me know the email id you use for git?

**Pull request merged**